test_regress/t/t_randomize.out

```diff
@@ -1,13 +1,9 @@
-%Warning-CONSTRAINTIGN: t/t_randomize.v:22:14: Constraint expression ignored (unsupported)
-                                             : ... note: In instance 't'
-   22 |       length dist { [0:1], [2:5] :/ 2, 6 := 6, 7 := 10, 1};
-      |              ^~~~
-                        ... For warning description see https://verilator.org/warn/CONSTRAINTIGN?v=latest
-                        ... Use "/* verilator lint_off CONSTRAINTIGN */" and lint_on around source to disable this message.
 %Warning-CONSTRAINTIGN: t/t_randomize.v:40:7: Constraint expression ignored (unsupported)
```

Comment:
As to your question, yes, if we are ignoring something, e.g. non-zero weights still, then we still need this or similar warning.

Reply:
Changed warning message to 'Constraint expression ignored (imperfect distribution)' here and in ConstraintBefore.